internal/app/cli.go

```diff
@@ -159,49 +147,44 @@ func init() {
 	for _, f := range files {
 		result, msg := fromFile(f, &fileState)
 		if result {
-			log.Printf(msg)
+			log.Info(msg)
 		} else {
-			logError(msg)
+			log.Fatal(msg)
 		}
 		// Merge Apps that already existed in the state
 		for appName, app := range fileState.Apps {
 			if _, ok := s.Apps[appName]; ok {
 				if err := mergo.Merge(s.Apps[appName], app, mergo.WithAppendSlice, mergo.WithOverride); err != nil {
-					logError("Failed to merge " + appName + " from desired state file" + f)
+					log.Fatal("Failed to merge " + appName + " from desired state file" + f)
 				}
 			}
 		}
 
 		// Merge the remaining Apps
 		if err := mergo.Merge(&s.Apps, &fileState.Apps); err != nil {
-			logError("Failed to merge desired state file" + f)
+			log.Fatal("Failed to merge desired state file" + f)
 		}
 		// All the apps are already merged, make fileState.Apps empty to avoid conflicts in the final merge
 		fileState.Apps = make(map[string]*release)
 
 		if err := mergo.Merge(&s, &fileState, mergo.WithAppendSlice, mergo.WithOverride); err != nil {
-			logError("Failed to merge desired state file" + f)
+			log.Fatal("Failed to merge desired state file" + f)
 		}
 	}
 
 	if debug {
 		s.print()
 	}
 
-	// validate that if we are running in tillerless mode, and we don't have the tiller plugin installed we should fail.
-	if !helmPluginExists("tiller") && s.Settings.Tillerless {
-		logError("ERROR: tillerless operation is specified and helm tiller plugin is not installed/configured correctly. Aborting!")
-	}
-
 	if !skipValidation {
 		// validate the desired state content
 		if len(files) > 0 {
-			if result, msg := s.validate(); !result { // syntax validation
-				logError(msg)
+			if err := s.validate(); err != nil { // syntax validation
+				log.Error(err.Error())
 			}
 		}
 	} else {
-		log.Println("INFO: desired state validation is skipped.")
+		log.Info("Desired state validation is skipped.")
 	}
 
 	if applyLabels {
```

Comment:
the `apply labels` option should be deprecated. It was introduced to make it possible to migrate from one version to another in v1.x. This should no longer be needed here.